example/WORKSPACE

```diff
@@ -60,28 +60,36 @@ http_archive(
 
 load("@rules_conda//:defs.bzl", "conda_create", "load_conda", "register_toolchain")
 
+USE_MAMBA = True
+
+CLEAN = True
```

Comment:
Please move the literal values back to rule calls. I know having these flags in variables enforces correctness and should be used in practice, but as this is just an example, I think it will be easier to understand for new users with literal values. Fewer abstractions -> less mental overhead. So for example `install_mamba = True` instead of `install_mamba=USE_MAMBA`.

Reply:
Ok, makes sense. I did that